### DISCUSSION TITLE
Using a ColorField as the trigger of a ColorPicker

### DISCUSSION THREAD

**Discussion by synecdokey** [adobe/react-spectrum#6558] [Q&A] [answered]
I'm trying to have a simple color field that opens the picker for more fine-grained control, how would I achieve that without resorting to a controlled component, and in a way that keeps it accessible?

**Comment by snowystinger** [comment, marked as answer]
I think you could join a colorfield with a color picker visually.
It could look a little like a combobox with the trigger button on the side. I don't think it could be supported as the same element because, in a combobox, the focus stays in the input. But for a component like this, focus would really need to move into the popover.

**Reply by synecdokey** [reply]
Yep, that makes a lot of sense. I suspected this, but now I can go back to design on my end and work that out. Thanks a lot!
